tests/selenium_tests.py

```diff
@@ -98,4 +96,4 @@ def tearDown(self) -> None:
 
 
 if __name__ == "__main__":
-    unittest.main()
\ No newline at end of file
+    unittest.main()
```

Comment:
one more related to base class:
```python
    def get_element(self, by_locator):
        webelement = self.explicitly_wait.until(expected_conditions.presence_of_element_located(by_locator),
                                   message=f"'{by_locator} element doesn't appear on the page"
                                   )
        return webelement
```
and
```python
    def fill(self, by_locator, value):
        self.explicitly_wait.until(expected_conditions.presence_of_element_located(by_locator),
                                   message=f"'{by_locator} element doesn't appear on the page"
                                   )
        self.driver.find_element(*by_locator).send_keys(value)
```
they have the same part of code so you can do sth like this:
```python
    def fill(self, by_locator, value):
        self.get_element(by_locator=by_locator).send_keys(value)
```